### PR TITLE
Add PPP Enze

### LINKED ISSUE #195: Draft Skeletal PPP (Enze)

### PR DESCRIPTION
**Pull request by sohenze**

### CODE REVIEW

File: docs/team/sohenze.md [hunk 1]
```diff
@@ -3,22 +3,106 @@ layout: page
...
-    - **Code contributed**: [RepoSense Link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=sohenze&breakdown=true)
+## Overview
 
-    - **Enhancements implemented**: to be added soon
+**TrackO** is a desktop application that helps small home-based business owners manage their orders and inventory efficiently
+in the form of an integrated solution built using Java with around 15 kLOC. The user interacts with the application via a
+CLI, and the application responds with its GUI, created with JavaFX.
 
-    - **Contributions to the UG**: to be added soon
+## Summary of Contributions
```
Comment (melissaharijanto) [inline docs/team/sohenze.md:12]: I think it's good practice to add a line break after the heading!

**Review by melissaharijanto — APPROVED** [review]

File: docs/team/sohenze.md [hunk 2]
```diff
@@ -3,22 +3,106 @@ layout: page
...
-    - **Contributions to the Developer Guide (Extracts)**: to be added soon
-
-    - **Contributions to the User Guide (Extracts)**: to be added soon
+* **Review/mentoring contributions:**:
+  * Provided alternative opinions to teammates' contributions
+    * Relevant pull request(s): [#105](https://github.com/AY2223S1-CS2103T-W15-3/tp/pull/105)
+  * Provided explanations to teammates regarding personal contributions
+    * Relevant pull request(s): [#123](https://github.com/AY2223S1-CS2103T-W15-3/tp/pull/123),
+    [#128](https://github.com/AY2223S1-CS2103T-W15-3/tp/pull/128)
+  * Assisted teammates in fixing bugs introduced after merging different pull requests
```
Comment (melissaharijanto) [inline docs/team/sohenze.md:105]: I think for this, can add the PR where you assisted me with editing the tests for `edito`!

Comment (sohenze) [reply docs/team/sohenze.md:105]: Yep added it in, thank you!

File: docs/team/sohenze.md (hunk 1 above)
Comment (sohenze) [reply docs/team/sohenze.md:12]: Thank you! Have made the changes!

**Review by Jiacheng-y — APPROVED** [review]

**Pull request merged**